frontend/src/components/GlobalAmount/GlobalAmount.test.tsx

```diff
@@ -0,0 +1,94 @@
+import React from 'react';
+import { shallow, mount } from 'enzyme';
+
+import { GlobalAmount } from './GlobalAmount';
+import { GlobalAmountPresentational } from './GlobalAmountPresentational';
+import { GlobalAmountPresentationalProps, GlobalAmountProps } from './Props';
+
+describe('Global Amount component', () => {
+    let props_pres: GlobalAmountPresentationalProps;
+    let props_cont: GlobalAmountProps;
+
+    beforeEach(() => {
+        props_pres = {
+            total: 0
+        };
+
+        props_cont = {
+            bank_accounts: [],
+            wallets: [],
+            cards: []
+        };
+    });
+
+    describe("when there is no accounts", () => {
+        it("presentational renders just with a zero", () => {
+            const wrapper = shallow(<GlobalAmountPresentational {...props_pres} />);
+            expect(wrapper.find(".main-amount").text()).toMatch(
+                "$0.00"
+            );
+        });
+        it("container sends justa a zero", () => {
+            const wrapper = mount(<GlobalAmount {...props_cont}/>);
```

Comment:
Codacy found an issue: [Identifier 'props_cont' is not in camel case.](https://app.codacy.com/manual/lgruelas/finance/pullRequest?prid=5891730)